pkg/chunk/gcp/gcs_object_client.go

```diff
@@ -111,8 +111,10 @@ func (s *GCSObjectClient) PutObject(ctx context.Context, objectKey string, objec
 func (s *GCSObjectClient) List(ctx context.Context, prefix string) ([]chunk.StorageObject, []chunk.StorageCommonPrefix, error) {
 	var storageObjects []chunk.StorageObject
 	var commonPrefixes []chunk.StorageCommonPrefix
+	q := &storage.Query{Prefix: prefix, Delimiter: s.delimiter}
+	q.SetAttrSelection([]string{"name", "updated"})
```

Comment:
This should be:
```suggestion
    q.SetAttrSelection([]string{"Name", "Updated"})
``` 

`SetAttrSelection` method will check and map these names to ones expected by API.

Reply:
Good catch.